aiounifi/interfaces/wlans.py

```diff
@@ -28,4 +28,4 @@ async def disable(self, wlan: Wlan) -> TypedApiResponse:
 
     def generate_wlan_qr_code(self, wlan: Wlan) -> bytes:
         """Generate QR code based on WLAN properties."""
-        return wlan_qr_code(wlan.name, wlan.x_passphrase)
+        return wlan_qr_code(wlan.name, wlan.x_passphrase, hidden=wlan.hide_ssid if wlan.hide_ssid is not None else False)
```

Comment:
```suggestion
        return wlan_qr_code(wlan.name, wlan.x_passphrase, hidden=wlan.hide_ssid is True)
```
can it be simplified to this

Reply:
Thanks for the suggestion - I changed that in https://github.com/Kane610/aiounifi/pull/872/commits/6e62ec11320ba630d7724a07fe09b5b1ec836e9e